Lib/fontParts/base/lib.py

```diff
@@ -369,6 +375,19 @@ def keys(self) -> List[str]:
         """
         return super(BaseLib, self).keys()
 
+    def values(self) -> BaseValues[LibValueType]:
+        """View the lib's values.
```

Comment:
Same here `Return the lib's values`? I just think that "View" is going to be really confusing to most folks using the library